src/ngraph/frontend/onnx_import/utils/reshape.hpp

```diff
@@ -22,13 +22,31 @@ namespace ngraph
 {
     namespace onnx_import
     {
+        namespace utils
+        {
+            /**
+             * @brief Flatten the input tensor into a 2D matrix.
```

Comment:
Please use `///` and `\brief` style for doc.

Reply:
I see that style change (https://github.com/NervanaSystems/ngraph/pull/1508) is related only to copyright notice. So we may use this style.

Reply:
No, your previous request was correct, see #1503